src/js/utils/useGlobalNav.js

```diff
@@ -7,20 +7,30 @@ const appIds = ['insights', 'openshift', 'cost-management', 'migrations', 'subsc
 
 const useGlobalNav = () => {
   const [state, setState] = useState({
+    isOpen: false,
     apps: [],
     filteredApps: [],
     isLoaded: false,
   });
   const setFilteredApps = (filteredApps) => setState((prev) => ({ ...prev, filteredApps }));
+  const setIsOpen = (isOpen) => setState((prev) => ({ ...prev, isOpen }));
   useEffect(() => {
-    (async () => {
-      const navigationYml = await sourceOfTruth();
-      const appData = await getNavFromConfig(load(navigationYml), undefined);
-      setState({ apps: appIds.map((id) => appData[id]).filter((app) => !!app), filteredApps: appIds.map((id) => appData[id]), isLoaded: true });
-    })();
-  }, []);
+    if (state.isOpen === true && state.isLoaded === false) {
```

Comment:
Just a one small thing. Should `isOpen` also have 3rd state? If not I would delete `=== true` so it is not confusing.

Reply:
I am not sure why `x === true` would be confusing to anyone

Reply:
In most of our codebase we dont use `x === true` just  `if(x)`. So, it suggest an idea that there might be third state that we care about. Exactly like `isLoaded` here. Which is probably not true for `isOpen`. it is a small thing and I don't really care about it. It was just thought that came to my mind when I was reading the code.